### ISSUE TITLE
Duplicating Display in Windows Settings Breaks LBM

### ISSUE THREAD

**Issue by Dustflinger** [mgth/LittleBigMouse#222]
I've been using LBM for around just over 2 years now, and honestly cannot live without it anymore. I run a 4k monitor on each side of my 1440p monitor, and everything has been great thus far! Seamless mouse across all three displays is super nice.

I recently just altered my set-up to include an output to my HDMI matrix so I can display my primary display on our TV or in the bedroom so that my spouse can watch while I play games. I am using the duplicate display option in Windows' display settings to clone my 1440p to this output. This caused LBM to simply cease correct function. No amount of tinkering allows me to regain LBM functionality on my rightmost monitor despite it showing in the GUI, and so long as LBM is enabled it will not allow me to mouse over to it. I can get proper alignment between my left and center monitor by hiding the duplicated display behind the true center display, yet no matter where I place any of the displays in LBM, it will not allow any traversal to the right monitor. When I restart LBM, my right monitor ends up hidden behind the center monitor.

Additionally, I was wondering if there was a way to simply ignore duplicated displays.

**Comment by Flanpit** [comment]
I have the same issue, if there was a way to just disable the mouse pointer for specific displays I wouldn't have to worry about this being an issue anymore. 

Right now I have the TV above my monitor so when I'm duplicating my monitor to the TV and bring my pointer to the top of it, the pointer teleports to the bottom of my monitor. Incredibly aggravating when trying to click the close window button or minimize button.

**Comment by Razwerkz** [comment]
Yeah, for sure LBM isn't handling mirrored displays properly.  I ended up throwing my mirror waaaaay off to the south-west, and making sure the maximum cursor travel was set short enough to not be able to reach.

**Comment by Big-Geek** [comment]
Thank you Razwerkz. That helped find something that worked for me. So far, this is working, though I won't be surprised if some weird behavior arises.

Similar to Razwerkz, I just manipulated the settings. I changed the monitor border on the mirrored screen (Wacom) from 20 units to 2000 units. Now, the mouse never travels to the mirrored screen. If I need to change it back, I simply can go to the settings for the mirroring monitor and change it back. 

This is the setup. I added 2 zeros as it should be easy to reverse this even with tiny text.

![LittleBigMouse_Wacom_Setup_2000](https://github.com/user-attachments/assets/296743aa-4e5e-4bf7-8981-56eac3812b69)

Here's what it looks like with the monitor in relation to the others. Prior to this change, it was butted up to the left side of the Samsung monitor. Now, it floats far up above and never gets any mouse travel. 

![LittleBigMouse_MonitorBorder](https://github.com/user-attachments/assets/8d299b9f-7586-4dbc-86c6-15ff04a9751f)

**Comment by magste123** [comment]
I have the same issue. I have 2 screens, main screen 1 and laptop screen 2. I'm duplicating laptop screen (2) to a tablet used as an input device. LBM sees this as three screens. No matter how I set the tablet screen parameters up in LBM, the scrolling is confused.  If I have the LBM confiig screen on screen 1 and start it, I can move the cursor to screen 2, where it gets stuck. I cannot move from screen 2 back to screen 1. Occationally it jumps back but it is seldom and random. I have to disconnect the tablet for it to work.

It would be good to be able to disable the tablet from LBM altogether - e.g tell LBM to igone one of the mirrored screens.

**Comment by mgth** [comment]
👋 **[LittleBigMouse 5.4.0](https://github.com/mgth/LittleBigMouse/releases/tag/v5.4.0) is out and should fix this.**

Duplicated displays got a dedicated rework in 5.3.0: clones are now recognized and handled as a single surface instead of separate monitors, so switching a display to duplicate mode (your HDMI matrix case) should no longer break the layout.

Please re-test with 5.4.0 (it requires the .NET 10 Desktop Runtime x64) and post back either way.

**Comment by FISSI0N** [comment]
unfortunately on 5.6.0 I just had a similar issue: I have 2 screens stacked on top of each other with the main screen being duplicated to a third one.
the duplication wasnt detected by LBM and as such I was able to move my cursor along the same screen twice, even while it was properly cloned. Only setting up a barrier worked, despite already leaving a gap between the duplicated screen and the other

**Comment by mgth** [comment]
That's by design : lbm just remember to what screen you enter. But with action button, you can also disable one.

**Comment by Mar-vo** [comment]
> That's by design : lbm just remember to what screen you enter. But with action button, you can also disable one.

why would it ever do this clearly bugged behaviour by design? 🙁
